src/vrl/selectielijst/tests/test_command.py

```diff
@@ -0,0 +1,236 @@
+             'procesobject': 'De vastgestelde organisatie inrichting',
+             }
+        )
+
+    def test_prepare_resultaat_generiek(self):
+        """ test prepare_procestype function: all general cols + cols for generiek resultaat"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        self.assertDictEqual(
+            prepare_resultaat(self.raw),
+            {'proces_type': proces_type,
+             'generiek_resultaat': None,
+             'nummer': 1,
+             'naam': 'Ingericht',
+             'omschrijving': '',
+             'herkomst': 'Risicoanalyse',
+             'waardering': 'vernietigen',
+             'procestermijn': 'nihil',
+             'bewaartermijn': relativedelta(years=10),
+             'toelichting': 'Invoering nieuwe werkwijze',
+             'algemeen_bestuur_en_inrichting_organisatie': False,
+             'bedrijfsvoering_en_personeel': True,
+             'publieke_informatie_en_registratie': False,
+             'burgerzaken': False,
+             'veiligheid': False,
+             'verkeer_en_vervoer': False,
+             'economie': False,
+             'onderwijs': False,
+             'sport_cultuur_en_recreatie': False,
+             'sociaal_domein': False,
+             'volksgezonheid_en_milieu': False,
+             'vhrosv': False,
+             'heffen_belastingen': False,
+             'alle_taakgebieden': False,
+             'procestermijn_opmerking': '',
+             }
+        )
+
+    def test_prepare_resultaat_specifiek(self):
+        """ test prepare_procestype function: cols for specifiek resultaat"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        generiek_resultaat = Resultaat.objects.create(**prepare_resultaat(self.raw))
+        specifiek = prepare_resultaat(self.specifiek)
+        self.assertEqual(
+            specifiek['generiek_resultaat'],
+            generiek_resultaat
+        )
+        self.assertEqual(
+            specifiek['nummer'],
+            2
+        )
+
+    def test_read_xls_row_to_dict_read(self):
+        """ test read_xls_row_to_dict generator: on reading and converting to dict first row of data in xls file"""
+        sheet_iter = read_xls_row_to_dict(self.sheet)
+        first_row = next(sheet_iter)
+        self.assertDictEqual(
+            first_row,
+            self.raw
+        )
+
+    def test_command_no_argument(self):
+        """  test command: fails if no arguments are provided """
+        self.assertRaises(Exception, call_command)
+
+    def test_command_save_procestype(self):
+        """ test handle method: test if could be created model object after prepare_procestype function"""
+        proces_type = ProcesType.objects.create(**prepare_procestype(self.raw))
+        self.assert_(True)
```

Comment:
never seen this before - I assume you're not sure what to assert here?

I'd split this up a bit & make an explicit assertion on the thing persisting in the db, more like:

```python
    def test_command_save_procestype(self):
        procestype_data = prepare_procestype(self.raw)

        proces_type = ProcesType.objects.create(**procestype_data)

        self.assertIsNotNone(proces_type.pk)
```
again following the pattern of:
1. setup
2. call thing to be tested
3. assert

Reply:
my point was to check, if such testcases are run without any exception
I'll fix it